tools/check_cfg_backup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if ls dev/mongooseim_*/etc/ejabberd.cfg.bak 1> /dev/null 2>&1; then
+    echo "Unrestored config exists. Your actions:"
+    echo "1. Stop nodes"
+    echo "2. Run \"make devrel\" again to restore original configs"
```

Comment:
+1, there might be some old junk in dev nodes, and since we rebuild then removing dev nodes brings no extra cost.